src/cluster.c

```diff
@@ -2663,7 +2663,7 @@ void clusterReadHandler(connection *conn) {
 
         if (nread <= 0) {
             /* I/O error... */
-            serverLog(LL_DEBUG,"I/O error reading from node link: %s",
+            serverLog(LL_WARNING, "I/O error reading from node link: %s",
```

Comment:
I'm not really convinced about the loglevels. I don't think the IO erorrs or connect errors are very actionable, since they don't include the target nodes, and for reasonable sized cluster if a node briefly has a networking issue it might spam the log as everything reconnects.

Reply:
make sense, reverted it